pkgs/by-name/ed/edopro/package.nix

```diff
@@ -0,0 +1,320 @@
+      libvorbis
+      lua5_3
+      nlohmann_json
+      ocgcore
+      openal
+      SDL2
+      sqlite
+    ];
+
+    # nixpkgs' gcc stack currently appears to not support LTO
+    postPatch = ''
+      substituteInPlace premake5.lua \
+        --replace-fail 'flags "LinkTimeOptimization"' 'removeflags "LinkTimeOptimization"'
+    '';
+
+    preBuild = ''
+      premake5 gmake2 \
+        --architecture=${archLabel} \
+        --covers=\"${covers_url}\" \
+        --fields=\"${fields_url}\" \
+        --pics=\"${pics_url}\" \
+        --prebuilt-core="${lib.getLib ocgcore}/lib" \
+        --sound=sfml
+    '';
+
+    enableParallelBuilding = true;
+    env = {
+      # remove after release 40.1.4+
+      # https://discord.com/channels/170601678658076672/792223685112889344/1286043823293599785
+      CXXFLAGS = "-include cstdint";
+      LDFLAGS = "-I ${irrlicht-edopro}/include -L ${irrlicht-edopro}/bin";
+    };
+    buildFlags = [
+      "verbose=true"
+      "config=release_${archLabel}"
+      "ygopro"
+    ];
+    makeFlags = [
+      "-C"
+      "build"
+    ];
+
+    installPhase = ''
+      runHook preInstall
+
+      mkdir -p $out/bin
+      cp bin/${archLabel}/release/ygopro $out/bin
+
+      runHook postInstall
+    '';
+  };
+
+  edopro-script =
+    let
+      assetsToCopy = lib.concatStringsSep "," [
+        # Needed if we download files from ProjectIgnis' website or any https only website.
+        "cacert.pem"
+        "config"
+        "deck"
+        "COPYING.txt"
+        "expansions"
+        "lflists"
+        "notices"
+        "puzzles"
+        "fonts"
+        "script"
+        "skin"
+        "sound"
+        "textures"
+        "WindBot"
+      ];
+    in
+    writeShellScriptBin "edopro" ''
+      set -eu
+      EDOPRO_DIR="''${XDG_DATA_HOME:-$HOME/.local/share}/edopro"
+
+      if [ ! -d $EDOPRO_DIR ]; then
+          mkdir -p $EDOPRO_DIR
+          cp -r --no-preserve=all ${assets.src}/{${assetsToCopy}} $EDOPRO_DIR
+          chmod -R go-rwx $EDOPRO_DIR
+
+          rm $EDOPRO_DIR/config/io.github.edo9300.EDOPro.desktop.in
+      fi
+
+      export PATH=PATH:'${lib.makeBinPath [ mono ]}';
+      export LD_LIBRARY_PATH='${
+        lib.makeLibraryPath [
+          libGL
+          libX11
+          libxkbcommon
+          libXxf86vm
+          sqlite
+          wayland
+          egl-wayland
+        ]
+      }';
+
+      exec ${edopro}/bin/ygopro -C $EDOPRO_DIR $@
+    '';
+
+  edopro-desktop = runCommandLocal "io.github.edo9300.EDOPro.desktop" { } ''
+    cp ${assets.src}/config/io.github.edo9300.EDOPro.desktop.in desktop-template
+
+    sed '/Path=/d' -i desktop-template
+    sed 's/Exec=.*/Exec=EDOPro/' -i desktop-template
+    sed 's/Icon=.*/Icon=EDOPro/' -i desktop-template
+
+    install -D desktop-template $out/share/applications/io.github.edo9300.EDOPro.desktop
+  '';
+in
+symlinkJoin {
+  name = "edopro-application-${deps.edopro-version}";
+  version = deps.edopro-version;
+  paths = [
+    edopro
```

Comment:
This causes `$out/bin/ygopro` to be linked, which is the unwrapped binary. I think it should be fine to just not add this to the `symlinkJoin`, since `edopro-script` handles the call to it?

```suggestion
```